src/data/sql-quiz.ts

```diff
@@ -1,4 +1,57 @@
 const sqlQuiz = [
+  {
+    Question: "In SQL,How many Primary keys can be created on a table?",
+    Answer: "Only one",
+    Distractor1: "Infinte",
+    Distractor2: "Finitely many",
+    Distractor3: "Two",
+    Explanation:
+      "The major reason is because that is the definition of the primary key. A table can have multiple unique keys that identify each row, but only one primary key",
```

Comment:
let's remove this first sentence 

```suggestion
      "A table can have multiple unique keys that identify each row, but only one primary key.",
```